src/GLua/AG/PrettyPrint.ag

```diff
@@ -182,16 +186,26 @@ infixl 6 <->
 a <-> b | a == empty = b
         | b == empty = a
         | otherwise  = a <> zchr ' ' <> b
+
+-- Always 10, so there is a default for precedence
+infixr 5 $$$
+($$$) :: Int -> Int -> Int
+a $$$ b = if a > b then 10 else 10
+
 }
 
 
 attr AllStuff
     syn pretty :: Doc
     inh indent :: Int
+    inh neighborPrecedence :: Int
     syn isMultiline use {||} {False} :: Bool -- whether code is printed over multiple lines
     inh ppconf :: PrettyPrintConfig
     syn copy :: self
 
+attr ExprStuff
+    syn precedence use {$$$} {10} :: Int
```

Comment:
With the `$$$` function, nothing but 10 ever bubbles up, unless it's a direct child. I don't think that's what you want.

It seems like either `min` or `max` should be the right function to combine two precedence values.

Reply:
That actually was exactly what I was going for, making it so 10 is the default precedence value when it isn't defined